pages/content/posts/tpac-sevilla.md

```diff
@@ -0,0 +1,46 @@
+
+There were several things I truly enjoyed about chairing and attending TPAC sessions. I really appreciate how scribing was taken seriously. I also appreciated how to be added to the speaker queue, all one has to do is type "q+" in the IRC channels. As an introvert, it can be quite stressful to figure out the right moment to speak up in a room full of (incredibly smart) people. Thanks to this inclusive feature, I q+'ed myself, was acknowledged by the chair, and was able to ask my question or share my perspective. If you are interested in more mechanics of this, I found a blog post on the Bocoup blog: [How to scribe at TPAC](https://bocoup.com/blog/how-to-scribe-at-tpac).
+
+## More sessions
+
+I recently joined the [WebDX W3C community group](https://www.w3.org/community/webdx/) and attended breakout sessions related to WebDX:
+
+- [Interop project](https://www.w3.org/2023/09/13-interop-minutes.html), chaired by James Graham (Mozilla).
+- [Beyond Interop](https://www.w3.org/2023/09/13-beyond-interop-minutes.html), chaired by Kadir Topal (Google).
+- [WebDX research and spec development](https://www.w3.org/2023/09/13-webdx-minutes.html), chaired by François Daoust (W3C).
+- [Secure the Web Forward](https://www.w3.org/2023/09/13-secure-the-web-minutes.html), chaired by Daniel Appelquist (Synk).
+
+Open Web Docs will be collaborating with the WebDX group a lot more going forward. Documentation is a major aspect of Developer Experience! As such, the research and work this community group is doing is very relevant to OWD's mission. As part of the goal of improving developer experience through better documentation, I met with Daniel Beck and Kadir Topal (Google) to discuss the next steps for the OWD project ["Grouping web features"](https://github.com/openwebdocs/project/issues/169). Daniel also made a [video about web features](https://www.w3.org/2023/09/TPAC/demos/web-features.html).
+
+Daniel Appelquist's "Secure the Web Forward" session was also good preparation for the [Secure the Web Forward virtual workshop](https://www.w3.org/2023/03/secure-the-web-forward/agenda.html) happening September 26-28. We are looking forward to discussing web security documentation and security education with security experts. While talking to Arnaud Le Hors (IBM), I realized we should likely start our security docs project by outlining the contents for a "MDN guide to secure your website / web app". This workshop will be invaluable in reaching this project's goals. I'm very much looking forward to this workshop and I'm curious to see how the format will play out. Expect a blog post about it soon!
```

Comment:
nit: the word "forward" is repeated 3 times at the start of this paragraph. Maybe find a way to remove the third instance. E.g "We're very eager to start discussing web security ..."